etcd/ci/etcd.rake

```diff
@@ -0,0 +1,73 @@
+require 'ci/common'
+
+def etcd_version
+  ENV['FLAVOR_VERSION'] || 'latest'
+end
+
+def etcd_rootdir
+  "#{ENV['INTEGRATIONS_DIR']}/etcd_#{etcd_version}"
+end
+
+namespace :ci do
+  namespace :etcd do |flavor|
+    task before_install: ['ci:common:before_install'] do
+      sh %(docker stop dd-test-etcd || true)
+      sh %(docker rm dd-test-etcd || true)
+    end
+
+    task install: ['ci:common:install'] do
+      use_venv = in_venv
+      install_requirements('etcd/requirements.txt',
+                           "--cache-dir #{ENV['PIP_CACHE']}",
+                           "#{ENV['VOLATILE_DIR']}/ci.log", use_venv)
+      sh %(docker create --expose 4001 --expose 2379 -p 4001:4001 -p 2379:2379 --name dd-test-etcd quay.io/coreos/etcd:v2.0.5 -listen-client-urls http://0.0.0.0:2379,http://0.0.0.0:4001)
```

Comment:
we only need one of those ports here since both are client ports. I suggest 4001 as it's the newer one.